app/jobs/access_copy_job.rb

```diff
@@ -0,0 +1,24 @@
+# A job to apply work permissions to all contained files set
+#
+class AccessCopyJob < ActiveJob::Base
```

Comment:
@awead I stayed away from permissions since visibility is part of permissions (in my mind).  I could be fine with InheritAccess.